LSP-vtsls.sublime-commands

```diff
@@ -0,0 +1,55 @@
+[
+    {
+        "caption": "Preferences: LSP-vtsls Settings",
+        "command": "edit_settings",
+        "args": {
+            "base_file": "${packages}/LSP-vtsls/LSP-vtsls.sublime-settings",
+            "default": "// Settings in here override those in \"LSP-vtsls/LSP-vtsls.sublime-settings\"\n\n{\n\t$0\n}\n"
+        }
+    },
+    {
+        "caption": "LSP-vtsls: Organize Imports",
+        "command": "lsp_typescript_execute",
```

Comment:
Why is this referencing `lsp_typescript_execute` (this command comes from LSP-typescript)
It should probably reference lsp_vtsls_execute

Reply:
Same comment holds true for all commands in this file. Change `lsp_typescript_execute` to `lsp_vtsls_execute`